motor/core.py

```diff
@@ -511,6 +518,7 @@ class AgnosticDatabase(AgnosticBaseProperties):
     __delegate_class__ = Database
 
     __hash__ = DelegateMethod()
+    __bool__ = DelegateMethod()
```

Comment:
This implements the breaking change in PYTHON-808 for Motor 3.0, right? If so, could you make a note about this in the changelog?

Reply:
Done